src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -440,13 +434,22 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
                         content={PatientExams}
                       />
                     </div>
-                  ) : userSection == "admissions" ? (
-                    <div className="patientDetails__nested_content">
-                      <PatientDetailsContent
-                        title="Admissions"
-                        content={PatientAdmissions}
-                      />
-                    </div>
+                  ) : userSection === "visits" ? (
+                    patient?.data?.status === PatientDTOStatusEnum.O ? (
+                      <div className="patientDetails__nested_content">
+                        <PatientDetailsContent
+                          title="Visits"
+                          content={PatientOPD}
+                        />
+                      </div>
+                    ) : (
+                      <div className="patientDetails__nested_content">
+                        <PatientDetailsContent
+                          title="Visits"
+                          content={PatientVisit}
+                        />
+                      </div>
+                    )
```

Comment:
I can't understand why this components are printed inline and not inside the RouterTabs as same as the orher.

Reply:
Ok I see now, it's to avoid the shows of the tab when a tabs has only one item. But, I think you could move this logic inside the RouterTabs component instead of create those excepions. I think that if you are inside the RouterTabs component you could still avoid to show the tab if there is only one item.

Reply:
OK ! I'll do it !

Reply:
You're right! Maybe we could just remove the RouterTabs if we don't use it anymore and render the components inside the code.